frontend/src/__mocks__/mockServingRuntimeTemplateK8sResource.ts

```diff
@@ -92,3 +92,91 @@ export const mockTemplateK8sResource = ({
   ],
   parameters: [],
 });
+
+//using it in unit test
+export const mockTemplateK8sResourceWithoutKind = ({
+  name = 'test-model',
+  namespace = 'opendatahub',
+}: MockResourceConfigType) => ({
+  apiVersion: 'template.openshift.io/v1',
+  metadata: {
+    name: 'template-ar2pcc',
+    namespace,
+    uid: '31277020-b60a-40c9-91bc-5ee3e2bb25ec',
+    resourceVersion: '164740435',
```

Comment:
I'm not sure whether to keep this here or should I create a new file to have API response to use if for unit test